EdgeDeflector/resources/nsis_installer.nsi

```diff
@@ -1,72 +1,237 @@
 ; Installer for EdgeDeflector
 
+
+;--------------------------------
+;Include Modern UI
+
+  !include "MUI2.nsh"
+
 ;--------------------------------
+;General
+
+  ;Define name of the product
+  !define PRODUCT "EdgeDeflector"
+  !define ABOUTURL "https://github.com/da2x/EdgeDeflector"
+  !define INSTALLATIONURL "https://github.com/da2x/EdgeDeflector/wiki/Thank-you-for-installing-EdgeDeflector"
+
+  ;Properly display all languages
+  Unicode true
+
+  ;Show 'console' in installer and uninstaller
+  ShowInstDetails "show"
+  ShowUninstDetails "show"
+
+  ;Name and file
+  Name "${PRODUCT}"
+  Icon "..\Icon.ico"
+  LicenseData "..\..\LICENSE"
+  OutFile "..\bin\Release\${PRODUCT}_install.exe"
 
-!define PRODUCT "EdgeDeflector"
+  ;Version
+  !define VERSIONMAJOR 1
```

Comment:
There are no version numbers in the current installer because I don’t want to update them. It would just complicate the process and add more work when releasing.

I’d only accept this change if they were sourced from [Properties/AssemblyInfo.cs](https://github.com/da2x/EdgeDeflector/blob/master/EdgeDeflector/Properties/AssemblyInfo.cs) without having to be independently updated in a separate file.

Reply:
OK, then I'll delete them because I have no idea how to do this simply without creating another script + it doesn't seem really important.